recipes/libtree/meta.yaml

```diff
@@ -1,47 +0,0 @@
-{% set name = "libtree" %}
```

Comment:
This needs to be put back as mentioned in https://github.com/sugatoray/staged-recipes/commit/5fe01d17182e0fc2cee4897a8f96991ac54909da

Reply:
Thank you, for the review. I have a question though:

The `main` branch had `libtree` in it when I created a new branch from it. However, now the `main` branch does not have libtree anymore. So, when I push my branch into main branch, git will see that there are only changes to add more files and not to remove. Right? Then, why would it still be necessary to put back `libtree` in there?

Reply:
@BastianZim I have applied all the changes suggested.

Reply:
I’m not 100% familiar with that part so I cannot fully comment on that but it’s conda-forge best practice. Additionally, it makes it easier for reviewers.

Reply:
Ok. Best practice it is then! :)
Thank you.

I guess, the bot would see that there exists a `libtree-feedstock` and hence it will disregard the libtree folder and its contents. However, I am assuming that you cannot submit a recipe if a feedstock already exists -- _am I correct to assume this?_

Reply:
The important thing to follow here is *don't touch files outside the recipe(s) you yourself are adding in your PR*. That's it, that's the rule.